### PR TITLE
Suppress resource path changes by default

### PR DESCRIPTION
**Pull request by cpsievert**
See #2584 (introduced by https://github.com/rstudio/shiny/pull/2459)

### CODE REVIEW

File: R/server.R [hunk 1]
```diff
@@ -92,7 +92,8 @@ addResourcePath <- function(prefix, directoryPath) {
   # Note that previous versions of shiny used to warn about this case,
   # but it was eventually removed since it caused confusion (#567).
   # It seems a good comprimise is to throw a more information message.
-  if (prefix %in% names(.globals$resourcePaths)) {
+  if (getOption("shiny.resourcePathChanges", FALSE) &&
```
Comment (wch) [inline R/server.R]: For options we generally separate words with `.`

Comment (wch) [reply R/server.R]: Although now that I look at it, we're pretty inconsistent with it :/

**Comment by wch** [comment]
This option seems so specific that it seems unlikely that someone will realize to turn it if they have problems. Maybe we should have a more general option that enables various diagnostic/debugging messages. But we could defer that discussion until the after the upcoming release.

In the short term we do need to change the default behavior so that it's quieter.

File: R/server.R (hunk 1 above)
Comment (cpsievert) [reply R/server.R]: Yea, I went with camel case since this is about `addResourcePath()`

**Comment by cpsievert** [comment]
Ok, so maybe I'll comment this whole thing out for the time being?

https://github.com/rstudio/shiny/blob/1c8071a96fbc026fd8c6738b64cc4d6e6a7905ed/R/server.R#L95-L106

**Comment by jcheng5** [comment]
I wonder if it'd make sense to have `createWebDependency` use a different code path? Perhaps even using a reserved URL prefix. We're unlikely to care about `createWebDependency` collisions, I think? It just means a new/different `jquery-1.1` or whatever is being provided.

**Pull request merged**